spot_rl_experiments/spot_rl/examples/run_pick.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
```

Comment:
Makes sense, this seems like a nice possible next task.
This can be tracked in the issue - https://github.com/facebookresearch/spot-sim2real/issues/137